dart/test/http_client/failed_request_client_test.dart

```diff
@@ -253,13 +253,13 @@ class Fixture {
     bool sendDefaultPii = true,
   }) {
     final mc = client ?? getClient();
+    var hub = _hub..options.sendDefaultPii = sendDefaultPii;
```

Comment:
Isn't this wrong as it sets the default value on the options?

Reply:
yes, sorry I forgot to change this, It should be
`var hub = _hub;
hub.options.sendDefaultPii = sendDefaultPii;`
I will fix this right away

Reply:
Ah nevermind, this is the test. Sorry!